app/scicrunch_processing_common.py

```diff
@@ -16,6 +16,7 @@
 VERSION = 'version'
 README = 'readme'
 TITLE = 'title'
+PUBLISH_DATE = 'publishDate'
```

Comment:
If this is intended to get through the processed results it needs to be added to PASS_THROUGH_KEYS.

Reply:
Yeah I thought that might be the case